app/component/board.js

```diff
@@ -43,7 +43,7 @@ export default class Board extends Component {
           }
           rows.push(
               <Row key={targeti}>
-                  <TargetProgress label={target.type}
+                  <TargetProgress label={target.getLabel()}
                                 hits={target.hits()}
                                 cleared={target.getDoneRound()}
```

Comment:
When I saw "cleared" actually used, I expected it to be a boolean (plus it was actually kinda being used that way). But actually it's the round that this target was cleared. This variable could be (heh) clearer. Like maybe ... "clearedOnRound" or maybe just "doneRound" for consistency.

Reply:
changed to `closedRound`, as closed is the proper cricket term